docs/Sysadmin-Guide.rst

```diff
@@ -103,14 +103,15 @@ written as an environment variable. For a Cloud Foundry application this could l
     ---
       applications:
       - name: standalone-uaa-cf-war
-        memory: 512M
+        memory: 4096M
```

Comment:
Any specific reason increasing this example memory size?

Reply:
our experience, ok we can have 2048 ? because I am sure that we 512 the current UAA with all the spring stuff inside will often crash with out of memory ....

What would you propose ?

Reply:
> What would you propose ?

Nothing really. I was just curious about how you picked the number. I do agrees that 512M was too low. The new value is fine with me.